compiler/luci/pass/src/RemoveRedundantTranspose.test.cpp

```diff
@@ -117,15 +232,17 @@ TEST(RemoveRedundantTransposePass, remove_consecutive_transpose_function_type1)
   while (pass.run(graph.get()))
     ;
   luci::CircleTranspose *transpose_node = nullptr;
+  int count = 0;
   for (auto node : loco::active_nodes(loco::output_nodes(graph.get())))
   {
     auto trans = dynamic_cast<luci::CircleTranspose *>(node);
     if (not trans)
       continue;
     transpose_node = trans;
-    break;
+    count++;
   }
   // No transpose node is in graph.
+  ASSERT_EQ(0, count);
```

Comment:
I think this is not needed for `no transpose node`

Reply:
Oh I will remove this.